specifyweb/specify/views.py

```diff
@@ -458,15 +458,15 @@ def resolve_record_merge_response(start_function, silent=True):
 # Only use if it can be assured that no constraints will be raised, requiring recursive merging.
 MERGING_OPTIMIZATION_TABLES = {
     'agent': {'spauditlog'},
-    'taxon': {'agent'},
-    'collectionobject' : {'agent'}
+    'agent': {'taxon'},
+    'agent' : {'collectionobject'}
```

Comment:
are you allowed to have two keys with the same value in python?

Reply:
I think that's an old commit